### PR TITLE
feat(courseoutlinestatus): improve alignment with platform highlights

### PR DESCRIPTION
**Pull request by MichaelRoytman**

### CODE REVIEW

**Comment by fysheets** [comment]
Quick rebase please :)

**Comment by MichaelRoytman** [comment]
@fysheets Rebased!

File: src/components/CourseOutlineStatus/index.jsx [hunk 1]
```diff
@@ -90,6 +93,74 @@ export default class CourseOutlineStatus extends React.Component {
     }
   }
 
+  onCourseChecklistHyperlinkClick = () => {
```
Comment (fysheets) [inline src/components/CourseOutlineStatus/index.jsx]: I like that you moved this into it's own function :)

File: src/components/CourseOutlineStatus/index.jsx [hunk 2]
```diff
@@ -159,28 +224,20 @@ export default class CourseOutlineStatus extends React.Component {
...
-          </h2>
-          <div>
-            {this.getAriaLiveRegion()}
-            {this.getBody()}
-          </div>
+        {this.getAriaLiveRegion()}
+        <div className="row no-gutters">
+          {this.getCourseStartDateStatus()}
+          {this.getCoursePacingTypeStatus()}
+          {this.getCourseChecklistStatus()}
```
Comment (fysheets) [inline src/components/CourseOutlineStatus/index.jsx]: I know this won't always be the correct thing to do, but something about the ease of reading what gets rendered in this render function feels really good.

Comment (MichaelRoytman) [reply src/components/CourseOutlineStatus/index.jsx]: Yay! :)

**Pull request merged**